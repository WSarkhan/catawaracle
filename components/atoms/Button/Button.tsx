import { View } from 'react-native';

interface Props {}

export const Button = (props: Props) => <View {...props} />;
