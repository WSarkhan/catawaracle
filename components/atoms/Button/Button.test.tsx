import * as React from 'react';
import * as renderer from 'react-test-renderer';
import { Button } from './Button';

describe('Button', () => {
  it('should match a snapshot', () => {
    const tree = renderer
      .create(<Button label="Title" onPress={() => {}} disabled={false} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
