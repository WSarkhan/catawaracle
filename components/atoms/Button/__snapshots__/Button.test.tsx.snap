// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should match a snapshot 1`] = `<View />`;
